### PR TITLE
changeTime check if time is Number instead of null

### LINKED ISSUE #81: [BUG] - ngx-mat-timepicker-field does not fire timechange when starting from 00:00
If ngx-mat-timepicker-field start from `00:00` timechange is not fired, it only works when hour and minutes are greater than zero or starting from one.

You can reproduce it here:
https://stackblitz.com/edit/uomhvz-sbpbeu?file=src%2Fexample%2Fcard-overview-example.ts

I will open a PR for the fix. It was pretty simple to solved.

### PR DESCRIPTION
**Pull request by betorobson**
it fixes #81

### CODE REVIEW

File: projects/ngx-mat-timepicker-repo/src/app/components/demo/demo.component.html [hunk 1]
```diff
@@ -230,10 +230,12 @@ <h2>ngx-mat-timepicker-field</h2>
                                               [max]="maxTime"
                                               [min]="minTime"
                                               [defaultTime]="'11:11 PM'"></ngx-mat-timepicker-field>
-                    <p><strong>format = 24</strong></p>
+                    <p><strong>format = 24 - debug: timepickerFieldValue = `{{timepickerFieldValue}}` </strong></p>
```
Comment (betorobson) [inline projects/ngx-mat-timepicker-repo/src/app/components/demo/demo.component.html:233]: only for debug propose

File: projects/ngx-mat-timepicker-repo/src/app/components/demo/demo.component.ts [hunk 2]
```diff
@@ -59,6 +59,7 @@ export class NgxMatTimepickerDemoComponent implements OnInit {
     selectedTime: string;
     selectedTimeFreeInput: string;
     selectedTimeWithRange: string;
+    timepickerFieldValue = '00:00';
```
Comment (betorobson) [inline projects/ngx-mat-timepicker-repo/src/app/components/demo/demo.component.ts:62]: only for debug propose

**Comment by tonysamperi** [comment]
Ok mate, I don't like your fix, but I see the point.
I'll do it myself.
For your entertainment => `isNaN` instead of `Number.isInteger`.

**Comment by rnogueira-namoadigital** [comment]
Ok mate, no worries, I was just wondering that isNaN can also return false to isNaN(null) diffrent from Number.isInteger(null).

**Comment by betorobson** [comment]
Ok mate, no worries, I was just wondering that isNaN can also return false to isNaN(null) diffrent from Number.isInteger(null).

* I answered before using my corporate account.

**Pull request closed without merging**